src/virtual_dom/vtag.rs

```diff
@@ -228,46 +332,56 @@ impl<CTX: 'static, COMP: Component<CTX>> VDiff for VTag<CTX, COMP> {
         sibling
     }
 
-    /// Renders virtual tag over DOM `Element`, but it also compares this with an opposite `VTag`
+    /// Renders virtual tag over DOM `Element`, but it also compares this with an ancestor `VTag`
     /// to compute what to patch in the actual DOM nodes.
+    ///
+    /// Note that this does NOT actually modify the DOM -- that is done FIXME WHERE???
```

Comment:
I have to remove this, this _does_ modify the DOM.

I still want to understand better when `self.reference` gets discarded. I'm a bit confused on that point.